adafruit_adxl37x.py

```diff
@@ -28,14 +28,33 @@
 
 """
 
+from struct import unpack
 from micropython import const
 import adafruit_adxl34x
 
+try:
+    from typing import Tuple, Optional
+
+    # This is only needed for typing
+    import busio  # pylint: disable=unused-import
```

Comment:
Is the `pylint` disable required?  Looks like the library is used so it should be okay without it.

Reply:
I copied it from the other lib, I'll try running it without.

Reply:
It passes without it! I'll remove it, let me know what to with the type hint so I can do a single commit.